returnn/tf/layers/base.py

```diff
@@ -1478,6 +1478,11 @@ def __init__(self, layer_class, layer_desc, **kwargs):
     self.layer_class_ = layer_class
     self.layer_desc = layer_desc
 
+    register_as_extern_data = layer_desc.get("register_as_extern_data")
```

Comment:
maybe add a comment here then (or in the class docstring) that it needs to manage the `register_as_extern_data`

Reply:
`register_as_extern_data` is already mentioned in the doc string of `DataNotAvailableLayer`, check if a comment is still necessary in the current code.